nf-test.config

```diff
@@ -10,4 +10,6 @@ config {
 
     // run all test with the defined docker profile from the main nextflow.config
     profile ""
-}
\ No newline at end of file
+
+    ignore [".venv/*"]
```

Comment:
It seems the ignore flag is not available in the current release of `nf-test` (0.8.4). When I try to run a test for a module, it outputs this error message: 
```
Error: Syntax errors in nf-test config file: groovy.lang.MissingPropertyException: No such property: ignore for class: com.askimed.nf.test.config.Config
```
I quickly went through the nf-test repo, and it seems the ignore was commited last month for the 0.9.0 release. I also might be wrong 😆